backend/sites/src/main.ts

```diff
@@ -10,4 +11,8 @@ async function main() {
   });
   await app.listen(process.env.PORT || 4007);
 }
-main();
+main().then(()=>{
+  Logger.log('Application started successfully')
```

Comment:
there is already a winston logger implemented in the application, we could replace with that as well

Reply:
code updated to use winston logger.